apicast/src/oauth/apicast_oauth.lua

```diff
@@ -34,25 +34,23 @@ end
 
 function _M.extract_params()
   local params = {}
-  local header_params = ngx.req.get_headers()
+  local auth
 
-  params.authorization = {}
-
-  if header_params['Authorization'] then
-    params.authorization = re.split(ngx.decode_base64(re.split(header_params['Authorization']," ", 'oj')[2]),":", 'oj')
```

Comment:
Why is the base64 decode function removed here? http_authorization module does this?